packages/victory-tooltip/src/index.d.ts

```diff
@@ -4,43 +4,41 @@ import {
   NumberOrCallback,
   StringOrNumberOrCallback,
   VictoryCommonProps,
+  VictoryLabelableProps,
   VictoryNumberCallback,
   VictoryThemeDefinition,
   VictoryStyleObject
 } from "victory-core";
 
-export interface VictoryTooltipProps {
+export interface VictoryTooltipProps extends VictoryCommonProps, VictoryLabelableProps {
```

Comment:
I don't think it's appropriate for VictoryTooltip to extend from VictoryCommonProps. There's a lot it won't use. I'd prefer keeping the few individual props that it does have in common (i.e. `width`, `height` etc) defined on this interface